cmd/atlas/internal/cmdext/cmdext.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2021-present The Atlas Authors. All rights reserved.
+// This source code is licensed under the Apache 2.0 license found
+// in the LICENSE file in the root directory of this source tree.
+
+// Package cmdext provides extensions to the Atlas CLI that
+// may be moved to a separate repository in the future.
+package cmdext
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	"ariga.io/atlas/schemahcl"
+	"ariga.io/atlas/sql/sqlclient"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/zclconf/go-cty/cty"
+	"gocloud.dev/runtimevar"
+	_ "gocloud.dev/runtimevar/awssecretsmanager"
+	_ "gocloud.dev/runtimevar/constantvar"
+	_ "gocloud.dev/runtimevar/gcpsecretmanager"
+	_ "gocloud.dev/runtimevar/httpvar"
+)
```

Comment:
Include all secrets? hashicorp/vault, azure?

Reply:
added filevar. blob depends on all other blob drivers, which I prefer to avoid atm.